src/main/ansible/vagrant.yml

```diff
@@ -55,42 +55,6 @@
       mode: "0775"
     with_items: "{{ bag_stores }}"
 
-  - name: Check whether the bags exist
```

Comment:
this removal is after the idea of @janvanmansum in https://github.com/DANS-KNAW/easy-bag-store/pull/48, which I fully support. However, would it make sense to have a symlink to the exact `/vagrant` path? What I mean is: the bags here were put in a certain `stores_dir`; would it make sense to make that a symlink there to `/vagrant/src/test/resources/vault/stores`? Similar for example for projects where we copy test data to from the host to the vm into the `vagrant_home` folder. Would it make sense in those cases to put a symlink to the test data on the host in that `vagrant_home` folder?

Curious to what you think, @janvanmansum and @jo-pol.

Reply:
Do you mean something like `/home/vagrant/test-bags` -> `/vagrant/src/test/resources/test-bags`? (Supposing that at that location there actually are test bags ;)

I wouldn't object to that. It would make it a bit easier to find them from the command line.

Reply:
that is precisely what I mean! Maybe @jo-pol can do that for this PR. I'll look into easy-bag-store.

Reply:
@jo-pol see this commit (https://github.com/DANS-KNAW/easy-bag-store/pull/52/commits/7778e29af78dc124191aa617f9d448441a2a6004) for an example of such a symlink